src/spurt/graph/utils.py

```diff
@@ -21,3 +31,18 @@ def order_points(p: tuple[int, int]) -> tuple[int, int]:
     if p[0] <= p[1]:
         return (p[0], p[1])
     return (p[1], p[0])
+
+
+def graph_laplacian(graph: GraphInterface) -> csc_matrix:
+    """Compute the graph laplacian."""
+    links = graph.links
+    nlinks = len(links)
+    data = np.ones(2 * nlinks, dtype=int)
+    data[0::2] = -1
+    amat = csr_matrix(
+        (
+            data,
+            (np.arange(2 * nlinks, dtype=int) // 2, links.flatten()),
+        )
+    )
```

Comment:
how about this to give more context for what `amat` is?

```suggestion
    # Build the incidence matrix of the graph:
    # Columns represent nodes, rows represent links (edges)
    # Each row has one -1 and one +1 for the source/dest. node index
    row_indices = np.arange(2 * nlinks, dtype=int) // 2
    col_indices = links.flatten()
    amat = csr_matrix((data, (row_indices, col_indices)))
```

Reply:
Done